_posts/2016-06-09-bayesian-quadrature.markdown

```diff
@@ -31,16 +31,89 @@ The actual integration that yields the distribution over the integral value is t
 of the integrand function, which is often analytic for certain choices of Gaussian processes, or 
 an analytic approximation (e.g., for WSABI [[3]](#refereces-on-quadrature)). 
 
-Thus, the success of Bayesian quadrature is based on three things: i) replacing an intractable integral with an inference 
+Thus, the success of Bayesian quadrature is based on three things: i) replacing an intractable integral with a regression 
 problem on the integrand function ii) replacing the actual integration with an easier, analytic integration of the surrogate model
 on the integrand function, and iii) actively choosing locations for integrand evaluations such that the budget is optimally used
 in the sense encoded by the acquisition scheme.
 
 #### Bayesian Quadrature in Emukit
-Emukit provides basic functionality for vanilla Bayesian quadrature [[1, 2]](#refereces-on-quadrature), 
-but also for more elaborate methods like WSABI [[3]](#refereces-on-quadrature). It alleviates the need to research and 
-implement the integration of the emulator, the acquisition scheme and the active loop while providing a frame to implement 
-and try out novel Bayesian quadrature methods.
+Emukit at the moment provides basic functionality for vanilla Bayesian quadrature where a GP surrogate model is placed upon 
+the integrand which is then integrated directly. 
+This is how it is done:
+
+First we define the function that we want to integrate, also called the *integrand*. Here we will chose the 1-dimensional
+Hennig1D function which is already implemented in Emukit. We also choose the integration bounds: a lower bound and an upper bound.
```

Comment:
can we add a link to a page/paper where a reader can visualize the function?

Reply:
I'll link the tutorial notebook